API/NoDocs/Microsoft.Extensions.Telemetry.Logging/ITagCollector.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Microsoft.Extensions.Telemetry.Abstractions'
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Compliance.Classification;
+
+namespace Microsoft.Extensions.Telemetry.Logging;
```

Comment:
@davidfowl @noahfalk 

Is the *Telemetry* namespace component useful for everything in this assembly? Or should all this stuff move into Microsoft.Extensions.Logging?

Reply:
Microsoft.Extensions.Logging makes sense to me. @tarekgh any thoughts here?

Reply:
Are we using any namespaces used by the runtime in the extensions in general? `Microsoft.Extensions.Logging` is reasonable to me too as long as not going to conflict with any of existing types.